sklearn/ensemble/tests/test_bagging.py

```diff
@@ -663,3 +663,28 @@ def test_oob_score_removed_on_warm_start():
     clf.fit(X, y)
 
     assert_raises(AttributeError, getattr, clf, "oob_score_")
+
+
+def test_consistent_index_sampling():
+    # Make sure randomly drawn indices generated at fit time are identically 
+    # reproduced at a later time (for use in, e.g., OOB scoring)
+    X, y = make_hastie_10_2(n_samples=20, random_state=1)
+    
+    bagging = BaggingClassifier(KNeighborsClassifier(), max_samples=0.5, max_features=0.5)
+    bagging.fit(X, y)
+
+    feat_inds_list = bagging.estimators_features_
+    feat_samp_inds_gen = bagging._get_estimators_data_draws()
```

Comment:
I still don't get why you can't use `estimators_samples_` here; or rather, why it's not equivalent to testing `_get_estimators_data_draws` (ignoring low-scale memory consumption during testing) except using public API.